packages/plugin-ext/src/main/browser/webview/webview.ts

```diff
@@ -49,6 +47,8 @@ import { WebviewResourceLoader } from '../../common/webview-protocol';
 import { WebviewResourceCache } from './webview-resource-cache';
 import { Endpoint } from '@theia/core/lib/browser/endpoint';
 
+const TRANSPARENT_OVERLAY_STYLE = 'theia-mini-browser-transparent-overlay';
```

Comment:
ah, but it won't work :( style declaration comes from the mini browser extension

@kittaakos Could we rename the style and move it to the core? It would be generally useful for iframes. We use it already for webview and mini-browser iframes. What would be the good name?

Reply:
I was about to suggest the same thing, and that it should be a generic style moved to core to avoid the issue. Perhaps the style can be renamed as `theia-transparent-overlay` (loses the `mini-browser` in the name)?

Reply:
Ack, @vince-fugnitto please feel free to share the suggested changes for me to add or open a separate PR.

Alternatively I can switch this PR to a branch on Theia so we can collaborate?

Reply:
@thegecko sure, all I would do is move this to an appropriate stylesheet in `@theia/core`:

https://github.com/eclipse-theia/theia/blob/0f16fe89b928e67ddea08b637a4cce393dea1d85/packages/mini-browser/src/browser/style/index.css#L162-L170

Rename the class `theia-mini-browser-transparent-overlay` to `theia-transparent-overlay`, and rename the following as well (to `theia-transparent-overlay`):

https://github.com/eclipse-theia/theia/blob/0f16fe89b928e67ddea08b637a4cce393dea1d85/packages/mini-browser/src/browser/mini-browser-content-style.ts#L30

Reply:
Updated